src/__screenshot_tests__/private-skin-components-screenshot-test.tsx

```diff
@@ -1,6 +1,6 @@
 import {openStoryPage} from '../test-utils';
 
-const SKINS = ['Movistar', 'O2', 'Vivo-new', 'Telefonica', 'Blau'] as const;
+const SKINS = ['O2', 'Vivo-new', 'Telefonica', 'Blau', 'Tu'] as const;
```

Comment:
I think it makes no sense to have Movistar here because the rest of the tests use the Movistar brand as a base.

Reply:
Please, keep it